### DISCUSSION TITLE
Whisper dialogue favorite languages

### DISCUSSION THREAD

**Discussion by rRobis** [SubtitleEdit/subtitleedit#8423] [Ideas]
There could be an option in the settings, to pick only certain languages to show in the languages dropdown in the whisper dialogue.

Or alternatively, make a way to pick favorite ones, so that they would show at the top (like we have for the formats).

**Comment by niksedk** [comment]
SE already has a language filter... it's just NOT implemented everywhere... but now Whisper languages should work with the language filter too:
https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.5/SubtitleEditBeta.zip

**Reply by rRobis** [reply]
Thanks! 
Tested, seems to be working.

I think it would be nice, if, when showing all languages, the first option above all the languages, would be "Change language filters" (like when you already have language filters set), so that the user doesn't have to find out about this possibility through digging the options.


Also, if we're at it, maybe, we can also add the same possibility to the auto-translate? I guess it would be appropriate to have it there too.

**Reply by rRobis** [reply]
@niksedk 
I just noticed, that following Whisper dialogues do not have language filters in the latest beta, when running them **on non-empty lines:**
1. Waveform context menu --> Run Whisper on selected paragraph  (on non-empty line)
2. List view context menu --> Selected lines --> Audio --> Audio to text (Whisper)   (on non-empty line)

(I saw the commit for the auto-translate language filter, but it's probably not built into the latest beta, so can't test it yet)
